README.md

```diff
@@ -23,8 +23,16 @@ make delete
 
 ## Testing
 
-The `./test` directory contains files for testing the app and its interactions with other apps. Make sure to put up to date binaries in `test/elfs` before launching the tests.
-In a future update, the latests binaries will be fetched automatically from Github.
+The `./test` directory contains files for testing the app and its interactions with other apps. Make sure to put up to date binaries in `test/elfs/` before launching the tests.
+In a future update, all binaries will be fetched automatically from Github.
+
+For now, only the exchange binary needs to be generated. The following flags are needed for the tests to pass:
+
+```
+make TESTING=1 TEST_PUBLIC_KEY=1
+```
+
+To run the tests:
```

Comment:
+ add the fact that the exchange app binaries must be copied as `test/elfs/exchange_nanos.elf` and `test/elfs/exchange_nanox.elf`